core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -121,6 +121,12 @@ object ApiVersion {
       case _ => throw new IllegalArgumentException(s"Invalid message format version $recordVersion")
     }
   }
+
+  /**
+   * Get the minimum inter broker version which supports the ApiVersion API.
+   */
+  def minVersionForApiDiscovery: ApiVersion = KAFKA_0_10_0_IV1
```

Comment:
Should we move this to the `client` project? Do we need this information in the `client` project too?

Reply:
I don't think so. The client assumes compatibility with the ApiVersions API. We don't support compatibility with brokers which do not support it.